shared/linkify.ts

```diff
@@ -26,6 +28,10 @@ LinkifyIt.prototype.normalize = function normalize(match: NoSchemaMatch) {
 		match.noschema = true;
 	}
 
+	if (match.schema === "web+") {
+		match.schema = "web+" + match.url.slice(4).match(webplus_scheme_chars)![0];
+	}
+
```

Comment:
This doesn't need to be in the global function... you can add a normalize function to the protocol registration

Reply:
that causes an unfixable type error.

Reply:
what do you mean?

Reply:
due to the `noschema` property. the protocol registration provides an entire replacement to the global function, so you'd have to set `noschema` in the protocol registration normalize function too.